### PR TITLE
fix date bug issue

### PR DESCRIPTION
**Pull request by lauherkcambria**
#### Which User story does this relate to?
- https://www.pivotaltracker.com/n/projects/1662141/stories/152825436

#### Brief feature description
- fix date field not taking in numbers.

#### Does this PR block/effect other stories?
- no

#### Do any commands needs to be run in order to run the project with this PR?
- n0

#### Code coverage percent diff as a result of this PR?
- karma: 69, rspec: 83

### CODE REVIEW

File: test/javascript/rfa_forms/rfa01a_edit_view/relationBetweenApplicantField_tests.js [hunk 1]
```diff
@@ -55,7 +52,15 @@ describe('Verify relation between applicant', function () {
   })
 
   it('verify date_of_relationship', () => {
-    // figure out how to test, react widgets datepicker testing is broke
+    let dateOfRelationshipField = relationCardComp.find('#date_of_relationship')
+    dateOfRelationshipField.simulate('change', {target: {value: '01/01/2000'}})
+    expect(setParentStateSpy).toHaveBeenCalledWith('date_of_relationship', '2000-01-01')
+  })
+
+  it('verify date_of_relationship', () => {
```
Comment (ratneshraval) [inline test/javascript/rfa_forms/rfa01a_edit_view/relationBetweenApplicantField_tests.js]: Can you update the text here so we know exactly what failed if/when it fails.

**Comment by ratneshraval** [comment]
@lauherkcambria Can you bump up coverage in karma.conf since this branch coverage is higher.

File: test/javascript/rfa_forms/rfa01a_edit_view/relationBetweenApplicantField_tests.js (hunk 1 above)
Comment (lauherkcambria) [reply test/javascript/rfa_forms/rfa01a_edit_view/relationBetweenApplicantField_tests.js]: sounds good, good catch

**Comment by lavanyareddyreddy** [comment]
Looks like this  PR needs one more development merge.

**Pull request merged**